pkg/cmd/debug.go

```diff
@@ -100,15 +100,15 @@ func (o *debugCmdOptions) run(cmd *cobra.Command, args []string) error {
 			// Context canceled
 			return
 		}
-		fmt.Printf("Disabling debug mode on integration %q\n", name)
+		fmt.Fprintf(cmd.OutOrStdout(), "Disabling debug mode on integration %q\n", name)
```

Comment:
`Fprintln` can be used here also.

Reply:
I did miss some of those Fprintf statements. I'll try to get them all this time. If I may ask, given the purpose of my next commit, would it not be better if do a commit amend and force push rather than do a new commit?

Reply:
No worries. I agree, amending and force push is a good idea.